pkg/instance/manager.go

```diff
@@ -26,35 +30,89 @@ const (
 	ManagerBlockedURL
 )
 
-// ManagerURL returns an external string for the given ManagerURL kind.
-func (i *Instance) ManagerURL(k ManagerURLKind) (s string, ok bool) {
-	defer func() {
-		if !ok {
-			s = i.PageURL("/manager_url_is_not_specified", nil)
-		}
-	}()
+type managerConfig struct {
+	Api *struct {
+		Url   string
```

Comment:
Can you use `API` and `URL` like it's the convention in go? Cf https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Done